package.json

```diff
@@ -181,7 +183,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm run prebuild:lang && git add lang/ && lint-staged"
```

Comment:
We should probably change `lint-staged` here rather than the husky hook. Should be a copy and past from https://github.com/videojs/video.js/pull/5649/files#r304555387

Reply:
I tried but it didn't want to work. Will try again.